test-packages/e2e-tests/test/mail/mail.spec.ts

```diff
@@ -29,15 +32,15 @@ describe('Mail', () => {
         );
       })
     ).toBe(true);
-  });
+  }, 60000);
```

Comment:
[q] Looks like the trafic of the local server is stored to some files, which are asserted on. Make sure the test directry is on the gitignore list and also stuff is cleaned up afterwards. Best would be that the mock FS is switched on for this folder and then the content disappears automatically.

Reply:
mock fs won't work for this e2e tests scenario, where we have client and server separations.